_posts/2025-07-23-cockpit-343.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 343
+author: tmatus
+date: "2025-07-23"
+tags: cockpit, machines
+slug: cockpit-343
+category: release
+summary: "Improved VNC and consoles, various UX improvements."
```

Comment:
I'd drop the "various UX improvements", as both topics are aready covered by "vnc". Also, "VNC" is a console.

```suggestion
summary: "Improved VNC and serial consoles"
```